src/status_im/common/standard_authentication/standard_auth/authorize.cljs

```diff
@@ -53,10 +53,12 @@
                                               (password-login {:on-press-biometrics
                                                                #(on-press-biometrics
                                                                  on-press-biometrics)}))}))]
-    (biometric/get-supported-type
-     (fn [biometric-type]
-       (if (and biometric-auth? biometric-type)
-         (biometrics-login biometrics-login)
-         (do
-           (reset-password)
-           (password-login {})))))))
+    (if biometric-auth?
+      (biometric/get-supported-type
+       (fn [biometric-type]
+         (if biometric-type
```

Comment:
what is biometric type btw? this code reads strangely imo

Reply:
I think you wrote it :rofl: 
![blame](https://github.com/status-im/status-mobile/assets/21865759/f9c99596-3826-465b-bf3e-d9bd30feaa3a)

It's just `:touch-id/:face-id/:fingerprint` or smth like this. I just added the check to test if biometrics is used for authorization (i.e. we're passing `biometric-auth?`) so we don't do unnecessary calls. For instance, for the settings' biometrics toggle we don't use biometrics for `authorize` since we specifically need the password from the user.

Reply:
Sort of, initially it was an external candidate that I merged the code for.
anyway - I more meant that previously it was 
If biometric-auth? And biometric type.
if biometric-type alone reads like something that we need to check which type is it? Perhaps wrapping in (boolean ) fn will be clearer 
Np either way 👍

![image](https://github.com/status-im/status-mobile/assets/22799766/795ea73f-702a-4bbb-a2c2-500d86312448)